packages/strapi-database/lib/migration-manager.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const debug = require('debug')('strapi-database:migration');
+const { isFunction } = require('lodash/fp');
+
+class MigrationManager {
+  constructor(db) {
+    debug('Initialize migration manager');
+    this.db = db;
+    this.migrations = [];
+  }
+
+  register(migration) {
+    debug('Register migration');
+    this.migrations.push(migration);
+  }
+
+  async runMigration(fn, options, context = {}) {
+    debug('Run migration');
+    await this.runBefore(options, context);
+    await await fn(options, context);
```

Comment:
What's the point of this double `await` since `migrateSchema` doesn't return anything? Can't you just put a single await here?

Reply:
just a typo it makes not sense